src/Utility/Tensor.C

```diff
@@ -710,6 +710,18 @@ Tensor operator* (const Tensor& A, const Tensor& B)
 }
 
 
+Tensor operator*(const double c, const Tensor& T)
+{
+  Tensor C(T.n);
+
+  for (int i = 1; i <= C.n; ++i)
+    for (int j = 1; j <= C.n; ++j)
+      C(i,j) = c*T(i,j);
+
+  return C;
+}
+
+
```

Comment:
Since we already have the *= operator defined, this can be done better(?) just like this:
{
  Tensor C(T);
  return C *= c;
}

and use Real instead of double to be consistent with the rest of this class.